theme/static/js/hs-vue/notifications-app.js

```diff
@@ -154,8 +153,8 @@ $(document).ready(function () {
                 }
             },
             canBeAborted: function (task) {
-                return task.status === 'In progress'
-                    || task.status === 'Pending execution'
+                return task.name != 'resource delete' && (task.status === 'In progress'
```

Comment:
I don't think we should be task names here.  Rather, I think it makes more sense to check whether a task can be aborted perhaps by whether they have the Aborted status or not.

Reply:
@sblack-usu task name has been used to identify different tasks so far. The Aborted status is only set on task when the task is already aborted, so it cannot be used to check whether the task can be aborted or not. In this case, we don't want resource delete task to be aborted until we figure out the file and metadata atomic operation.